tests/rpc/rpc_01-system_test.go

```diff
@@ -113,41 +107,25 @@ func TestSystemRPC(t *testing.T) {
 		},
 	}
 
-	t.Log("going to start gossamer")
-
-	localPidList, err := StartNodes(t, make([]*exec.Cmd, 1))
+	t.Log("starting gossamer...")
+	nodes, err := utils.StartNodes(t, 3)
```

Comment:
I see you updated this to start 3 nodes here, but you've left it to start 1 node in the other tests, is there a reason for starting 3 here?

Reply:
this one has an expected peers of 2, so I started 3 nodes so that the node would have 2 peers

Reply:
This is because of the assertion for the SystemHealthResponse